src/plugins/unified_histogram/public/layout/hooks/use_lens_suggestions.ts

```diff
@@ -107,15 +106,18 @@ export const useLensSuggestions = ({
           },
         ] as DatatableColumn[],
         query: {
-          esql: histogramQuery,
+          esql: esqlQuery,
         },
       };
       const sug = lensSuggestionsApi(context, dataView, ['lnsDatatable']) ?? [];
       if (sug.length) {
+        histogramQuery.current = { esql: esqlQuery };
         return sug[0];
       }
+      histogramQuery.current = undefined;
       return undefined;
```

Comment:
these two lines are not necessary, are they? it would just follow line 120 and 121

Reply:
I will remove them